README.md

```diff
@@ -64,26 +64,47 @@ In the Julia REPL, first load the package Trixi
 ```julia
 julia> using Trixi
 ```
-Then, to start a simulation, execute
+Then start a simulation by executing
 ```julia
-Julia> Trixi.run(default_example())
+julia> Trixi.run(default_example())
 ```
-The `run(...)` method expects a single string argument with the path to a
+To visualize the results, load the package Trixi2Img
+```julia
+julia> using Trixi2Img
+```
+and generate a contour plot of the results with
+```julia
+julia> Trixi2Img.convert("out/solution_000040.h5", grid_lines=true) # On Windows, use "out\\solution_000040.h5"
```

Comment:
`joinpath("out", "solution_000040.h5")`

Reply:
TBH, I had it with `joinpath` at first but then decided to do it this way since the `joinpath` makes the command look much more complicated than it is. This in the spirit of giving the simplest explanation, I decided to hardcode it. But if you feel like this actually makes it more difficult to understand, I can also use the Julian approach.